frontend/src/core/resource/components/ResourceMenu.js

```diff
@@ -0,0 +1,172 @@
+/* @flow */
+
+import * as React from 'react';
+import { Localized } from 'fluent-react';
+import onClickOutside from 'react-onclickoutside';
+
+import './ResourceMenu.css';
+
+import { ResourceItem, ResourcePercent } from '..';
```

Comment:
You should import from the files directly, and not expose these outside this module.